packages/ag-charts-website/src/content/docs/benchmarks/_examples/summary/data.ts

```diff
@@ -922,5 +922,129 @@ export function getData() {
                 },
             },
         },
+        {
```

Comment:
Is this expected to be added?

Reply:
I think no. This was added when running `nx benchmark -c update`